### PR TITLE
Update ApiAnalyzer section

### PR DESCRIPTION
**Pull request by Lxiamail**
## Summary

to clarify the ApiAnalyzer tool's unique value and avoid including the duplicate content as in ApiAnalyzer doc.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/core/porting/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Port code from .NET Framework to .NET Core
 description: Understand the porting process and discover tools you may find helpful when porting a .NET Framework project to .NET Core.
 author: cartermp
-ms.date: 12/07/2018
+ms.date: 7/03/2019
```
Comment (mairaw) [inline docs/core/porting/index.md]:
```suggestion
ms.date: 07/03/2019
```

File: docs/core/porting/index.md [hunk 2]
```diff
@@ -34,15 +34,7 @@ This is the process we recommend you take when porting your project to .NET Core
...
-
-	-   Use of a .NET Standard API that throws [PlatformNotSupportedException](https://docs.microsoft.com/en-us/dotnet/api/system.platformnotsupportedexception) (PC001).
-	-   Use of a .NET Standard API that isn't available on the .NET Framework 4.6.1 (PC002).
-	-   Use of a native API that doesn’t exist in UWP (PC003).
-	-   Use of <xref:system.windows.forms.control.begininvoke> and <xref:system.windows.forms.control.endinvoke> APIs (PC004).
-	-   Use of an API that is marked as deprecated (DEXXXX).
-	
-
+* .NET API Analyzer - A Roslyn analyzer that discovers .NET Standard API that throws [PlatformNotSupportedException](https://docs.microsoft.com/en-us/dotnet/api/system.platformnotsupportedexception) on some platforms, detects calls to deprecated APIs and discovers some other potential compatibility risks for C# APIs on different platforms. For more information, see [.NET API analyzer](../../standard/analyzers/api-analyzer.md).  This analyzer is helpful after you already created your .NET Core project to identify runtime behavior differences on different platforms.
```
Comment (mairaw) [inline docs/core/porting/index.md]:
```suggestion
* .NET API Analyzer - A Roslyn analyzer that discovers .NET Standard API that throws <xref:System.PlatformNotSupportedException> on some platforms, detects calls to deprecated APIs, and discovers some other potential compatibility risks for C# APIs on different platforms. For more information, see [.NET API analyzer](../../standard/analyzers/api-analyzer.md). This analyzer is helpful after you already created your .NET Core project to identify runtime behavior differences on different platforms. 
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good @Lxiamail. Just a few quick comments to be addressed.

**Comment by Lxiamail** [comment]
> Looks good @Lxiamail. Just a few quick comments to be addressed.

Fixed all per feedback

**Review by mairaw — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**